ntoskrnl/include/internal/x86x64/ke.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * PROJECT:     ReactOS Kernel
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Architecture specific support routines shared between x86 and x64
+ * COPYRIGHT:   Copyright 2025 Timo Kreuzer <timo.kreuzer@reactos.org>
+ */
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+static const ULONG KxFlushIndividualProcessPagesMaximum = 33; // Based on Linux tlb_single_page_flush_ceiling
+static const ULONG KxFlushIndividualGlobalPagesMaximum = 100; // Just a guess
```

Comment:
What are these variables for? Some sort of arbitrary threshold that when reached in KeFlushRangeTb and friends, a full flush is done?

Reply:
Yes. Use the source, Luke.